src/test/java/org/example/api/RoleServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package org.example.api;
+
+import org.example.cli.Role;
+import org.example.client.DatabaseConnectionException;
+import org.example.client.FailedToGetRoleException;
+import org.example.db.DatabaseConnector;
+import org.example.db.RoleDao;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RoleServiceTest {
+    RoleDao roleDao = Mockito.mock(RoleDao.class);
+    DatabaseConnector databaseConnector = Mockito.mock(DatabaseConnector.class);
+
+    RoleService roleService = new RoleService(roleDao, databaseConnector);
+
+    Connection conn;
+
+    /*
+    Unit test for the getRoles method
+
+    When the dao returns a list of roles
+
+    Expect the list of roles to be returned
+
+    This should pass without code changes
+     */
+
+    @Test
+    void getAllRoles_shouldReturnRoleList() throws SQLException, FailedToGetRoleException, DatabaseConnectionException {
+        List<Role> expectedResult = new ArrayList<>();
+        Role role1 = new Role(1, "test","test","test","test","test","test");
+        Role role2 = new Role(2, "tester","tester","tester","tester","tester","tester");
```

Comment:
Can you make both `role1` and `role2` consistent with spacing between each of the values.

```
Role role1 = new Role(1, "test", "test"," test", "test", "test", "test");
Role role2 = new Role(2, "tester", "tester", "tester", "tester", "tester", "tester");
```